spec/facility_spec.rb

```diff
@@ -23,4 +38,198 @@
+      expect(@facility_1.administer_road_test(@registrant_3)).to eq(false)
+
+      @registrant_3.earn_permit
+      expect(@facility_1.administer_road_test(@registrant_3)).to eq(false)
+      expect(@registrant_3.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
+
+    it "adds Road Test to facility_1" do
+      @facility_1.add_service("Written Test")
+      @facility_1.administer_written_test(@registrant_1)
+      expect(@facility_1.administer_road_test(@registrant_1)).to eq(false)
+
+      @facility_1.add_service("Road Test")
+      expect(@facility_1.services).to eq(["Written Test", "Road Test"])
+
+      @facility_1.administer_road_test(@registrant_1)
+      expect(@registrant_1.license_data).to eq({:written=>true, :license=>true, :renewed=>false})
+    end
+
+    it "registrant_2 completes road test" do
+      @facility_1.add_service("Written Test")
+      @facility_1.add_service("Road Test")
+      @registrant_2.earn_permit
+      @facility_1.administer_written_test(@registrant_2)
+      @facility_1.administer_road_test(@registrant_2)
+
+      expect(@facility_1.administer_road_test(@registrant_2)).to eq(true)
+      expect(@registrant_2.license_data).to eq({:written=>true, :license=>true, :renewed=>false})
+    end
+  end
+
+  describe "Renew license" do
+    it "adds service to facility_1" do
+      @facility_1.add_service("Written Test")
+      @facility_1.add_service("Road Test")
+      @facility_1.administer_road_test(@registrant_1)
+      @facility_1.administer_road_test(@registrant_2)
+      @facility_1.administer_written_test(@registrant_1)
+      @facility_1.administer_written_test(@registrant_2)
+
+      expect(@facility_1.renew_drivers_license(@registrant_1)).to eq(false)
+
+      @facility_1.add_service("Renew License")
+      expect(@facility_1.services).to eq(["Written Test", "Road Test", "Renew License"])
+
+      @facility_1.renew_drivers_license(@registrant_1)
+      expect(@facility_1.renew_drivers_license(@registrant_1)).to eq(true)
+      expect(@registrant_1.license_data).to eq({:written=>true, :license=>true, :renewed=>true})
+    end
+
+    it "renews license for other registrants" do
+      @facility_1.add_service("Written Test")
+      @facility_1.add_service("Road Test")
+      @facility_1.add_service("Renew License")
+      @registrant_2.earn_permit
+      @facility_1.administer_road_test(@registrant_2)
+      @facility_1.administer_written_test(@registrant_2)
+
+      @facility_1.renew_drivers_license(@registrant_3)
+      expect(@facility_1.renew_drivers_license(@registrant_3)).to eq(false)
+      expect(@registrant_3.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+
+      @facility_2.renew_drivers_license(@registrant_2)
+      expect(@facility_1.renew_drivers_license(@registrant_2)).to eq(true)
+      expect(@registrant_2.license_data).to eq({:written=>true, :license=>true, :renewed=>true})
+    end
+  end
+
```

Comment:
I don't see tests for some of your helper methods like `create_plate_type` that you use in the register_vehicle method. Make sure we are unit testing all of our methods, including helper methods.